omnigibson/prims/cloth_prim.py

```diff
@@ -200,14 +269,199 @@ def density(self):
     def density(self, density):
         raise NotImplementedError("Cannot set density for ClothPrim")
 
+    @property
+    def body_name(self):
+        """
+        Returns:
+            str: Name of this body
+        """
+        return self.prim_path.split("/")[-1]
+
+    def get_linear_velocity(self):
+        """
+        Returns:
+            np.ndarray: current average linear velocity of the particles of the cloth prim. Shape (3,).
+        """
+        return np.array(self._prim.GetAttribute("velocities").Get()).mean(axis=0)
+
+    def get_angular_velocity(self):
+        """
+        Returns:
+            np.ndarray: zero vector as a placeholder because a cloth prim doesn't have an angular velocity. Shape (3,).
+        """
+        return np.zeros(3)
```

Comment:
we could probably have some aggregate metric of angular velocity from individual particle velocities (e.g.: [HERE](https://arxiv.org/pdf/physics/0401146.pdf)), but probably not worth the effort?

Reply:
I think it's probably not worth the effort. Also, I am leaning towards not to do it because the object base link has no orientation change (obj.get_orientation() stays constant as simulation plays). It's better to have zero angular velocity to be consistent.